portfolio/src/main/webapp/css/main.css

```diff
@@ -0,0 +1,323 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;700&display=swap");
+body {
+  height: 100vh;
+  margin: 0;
+  font-family: 'Poppins', arial;
+}
+
+ul {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+}
+
+a {
+  text-decoration: none;
+}
+
+img {
+  width: 100%;
+}
+
+.navigation-container {
+  background: #1616AC;
+}
+
+.navigation-container a {
+  /* sass exclusive feature */
+  color: white;
+}
+
+header {
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-box-pack: justify;
+      -ms-flex-pack: justify;
+          justify-content: space-between;
+  /* This line makes header columns in two lines*/
+  padding: 2em;
+}
+
+header .logo {
+  font-weight: bold;
+}
+
+header ul {
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+}
+
+header ul a {
+  display: block;
+  padding: 0 1em;
+}
+
+.social-header, .hero-design {
```

Comment:
Separate selectors and declarations by new lines, see https://google.github.io/styleguide/htmlcssguide.html#Selector_and_Declaration_Separation

Reply:
Done! :)

Reply:
I think a new line is still needed?